docs/admin-entity.md

```diff
@@ -0,0 +1,69 @@
+## AdminEntity class
+
+You can register entities with options using the `AdminEntity` class
+
+The simpliest example of registering an `AdminEntity` class to for a `User` entity looks like this:
+
+```typescript
+// user.admin.ts
+import { AdminEntity, DefaultAdminSite } from 'nestjs-admin'
+import { UserAdmin } from './user.admin'
+import { User } from './user.entity'
+
+export class UserAdmin extends AdminEntity {
+  entity = User
+}
+
+@Module({...})
+export class UserModule {
+  constructor(private readonly adminSite: DefaultAdminSite) {
+    adminSite.register('User', UserAdmin)
+    ...
+  }
+}
+
+```
+
+This will behave the same as:
```

Comment:
```suggestion
A this point, without any configuration in the UserAdmin, it will behave the same as registering the User entity directly (`adminSite.register('User', User)`)
```

Reply:
Then you can ditch the snippet below